doc/conf.py

```diff
@@ -59,11 +59,11 @@
 
 html_theme_options = {
     'logo_only': True,  # if we have a html_logo below, this shows only the logo with no title text
-    # Toc options
+    # ToC options
     'collapse_navigation': False,
     'sticky_navigation': True,
 }
-html_logo = '_static/logo-negativ.svg'
```

Comment:
This file is renamed.

Note: if a given family of changes is objectionable, it's trivial for me to drop, just let me know.

Reply:
The name is in Czech, apparently.